controllers/jobs.js

```diff
@@ -0,0 +1,81 @@
+const Job = require('../models/Job');
+
+const getAllJobs = async(req, res) =>{
+    const jobs = await Job.find({createdBy:req.user._id});
+    res.render('jobs',{jobs});
+};
+
+const newJobForm = (req, res)=>{
```

Comment:
Not sure you want to use this exact variable name here and in the function below